### DISCUSSION TITLE
Running Tests with PostgreSQL with multiple schemas

### DISCUSSION THREAD

**Discussion by gsoareslima** [laravel/framework#51244] [Q&A]
Hey guys.

If anyone could help me I would be very grateful.

I'm trying to create tests with Pest and I'm getting the following error:

```
FAILED Tests\Feature\Admin\LoginTest > login page QueryException
SQLSTATE[42P07]: Duplicate table: 7 ERROR: relation "equipes" already exists (Connection: esus_bi, SQL: create table "esus"."equipes" ...
```

Currently I am using a database called 'testing' in PostgreSQL itself as a test bench, and I noticed that 'Illuminate\Foundation\Testing\RefreshDatabase' is not dropping tables that are in a different schema than public.

My database.php looks like this:
```
 'esus_bi' => [
            'driver' => 'pgsql',
            'host' => env('DB_HOST', '127.0.0.1'),
            'port' => env('DB_PORT', '5432'),
            'database' => env('DB_DATABASE'),
            'username' => env('DB_USERNAME'),
            'password' => env('DB_PASSWORD', ''),
            'charset' => 'utf8',
            'prefix' => '',
            'prefix_indexes' => true,
            'search_path' => 'public',
            'sslmode' => 'prefer',
],
```

And my migration:

```
Schema::create('esus.equipes', function (Blueprint $table) {
...
})
```

**Comment by zaharin** [comment]
To specify multiple PostgreSQL schemas in your `search_path`, update your database configuration as follows:

1. Open `config/database.php`
2. Locate your PostgreSQL connection settings
3. Add the `search_path` parameter with your schemas:

```php
'pgsql' => [
    'driver' => 'pgsql',
    'host' => env('DB_HOST', '127.0.0.1'),
    // ... other default settings ...
    'search_path' => 'public,esus', // Your solution here
    'sslmode' => 'prefer',
],
